targets/frontend/src/lib/azure.js

```diff
@@ -0,0 +1,87 @@
+import { AbortController } from "@azure/abort-controller";
+import {
+  BlobServiceClient,
+  StorageSharedKeyCredential,
+} from "@azure/storage-blob";
+import { getToken } from "src/lib/auth/token";
+
+const AZURE_STORAGE_ACCOUNT = {
+  key: process.env.AZURE_STORAGE_ACCOUNT_KEY || "",
+  name: process.env.AZURE_STORAGE_ACCOUNT_NAME || "",
+};
+
+export const uploadFiles = (container, formData) => {
```

Comment:
We could factorize these calls and I wonder if we should move that to the page component and leave in API code that will be use by server api

Reply:
techniquement c'est pas trop un component plutot dans lib alors ? Pour la facto bon je pense que ça peut attendre